docs/usage/web-api.md

```diff
@@ -55,6 +55,43 @@ important to ensure only authorized parties may access it, or other measures to
 especially in a future Mainnet deployment.
 ```
 
+### Notes on publisher operation
+
+We list here a few important details on how the publisher deals with funds and objects on Sui.
+
+#### Number of sub-wallets
+
+As mentioned above, the publisher uses sub-wallets to allow storing blobs in parallel. By default,
+the publisher uses 8 sub-wallets, meaning it can store 8 blobs at the same time.
+
+#### Funds in sub-wallets
+
+Each of the sub-wallets requires funds to interact with the chain and purchase storage. For this
+reason, a background process checks periodically if the sub-wallets have enough funds. In steady
+state, each of the sub-wallets will have a balance of 0.5-1.0 SUI and WAL. The amount and triggers
+for coin refills can be configured through CLI arguments.
+
+#### Lifecycle of created `Blob` objects
+
+Each store operation in Walrus creates a `Blob` object on Sui. This blob object represents the
+(partial) ownership over the associated data, and allows certain data management operations (e.g.,
+in the case of deletable blobs).
+
+When the publisher stores a blob on behalf of a client, the `Blob` object is initially owned by the
+sub-wallet that stored the blob. Then, the following cases are possible, depending on the
+configuration:
+
+- If the client requests to store a blob and specifies the `send_object_to` query parameter (see
+  [the relevant section](#store) for examples), then the `Blob` object is transferred to the
+  specified address. This is a way for clients to get back the created object for their data.
+- If the `send_object_to` query parameter is not specified, two cases are possible:
+  - if the publisher was run with the `--keep` flag, then the sub-wallet transfers the
+      newly-created blob object to the main wallet, such that all these objects are kept there.
+  - if the `--keep` flag was omitted, then the sub-wallet *immediately burns* the `Blob` object.
+      Since no one has requested the object, and the availability of the data on Walrus is
+      independent of the existence of such object, it is safe to do so. This is to avoid cluttering
+      the sub-wallet with many blob objects.
```

Comment:
Nits (capitalization and indentation):
```suggestion
  - If the publisher was run with the `--keep` flag, then the sub-wallet transfers the
    newly-created blob object to the main wallet, such that all these objects are kept there.
  - If the `--keep` flag was omitted, then the sub-wallet *immediately burns* the `Blob` object.
    Since no one has requested the object, and the availability of the data on Walrus is independent
    of the existence of such object, it is safe to do so. This is to avoid cluttering the sub-wallet
    with many blob objects.
```